atm_challenge/step_3.md

```diff
@@ -101,7 +101,7 @@ def withdraw(amount, account)
   # We will be using Ruby's `case`- `when` - `then` flow control statement
   # and check if there is enough funds in the account
   case
-  when amount > account.balance then
+  when amount > account.balance
```

Comment:
Just reiterating since it got buried by other changes, the `ATM` has `funds` and the `account` has a `balance` - I believe that's why things got confused in the first place.